pyproject.toml

```diff
@@ -207,10 +211,6 @@ py-version = "3.10"
 # source root.
 # source-roots =
 
-# When enabled, pylint would attempt to guess common misconfiguration and emit
-# user-friendly hints instead of false-positive error messages.
-suggestion-mode = true
-
```

Comment:
This was dropped because the option is no longer supported by pylint. (Version 4+ always behaves as if this option is on.)